source/api/extension/navplace/index.md

```diff
@@ -139,6 +145,8 @@ The `navPlace` property identifies a single or multiple geographic areas pertine
 *   The `navPlace` property _MUST NOT_ be used on other IIIF resource types.
```

Comment:
Do we do this in other specs? I wonder if we just say where it can be and don't mention when we can't. Does this conflict with "Define Success, Not Failure" from https://iiif.io/api/annex/notes/design_patterns/#define-success-not-failure

Reply:
Yes, we do this in other specs.  https://iiif.io/api/presentation/3.0/#navdate

Reply:
OK fair enough :-)

"Other types of resource must not have the navDate property."